tests/network/config_services.pm

```diff
@@ -0,0 +1,84 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test preparing services to use with multimachine scenarios.
+#  At least confiugre:
+#   - http - A basic http support with apache2
+#   - ldap - A simple openladp with users.
+#   - ftp  - simple ftp server
+#  Use on test variables settings: SUPPORT_SERVER_ROLES=http,ldap
+#  The firewall was disable and network configuration used was provide by  tests/network/setup_multimachine.pm
+#  This script was based from tests/support_server/setup.pm, but not use serial
+#  console or neddles. This is to facilites to run on all SLES/OPensuse versions.
+#
+# Maintainer: Marcelo Martins <mmartins@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use mm_network 'setup_static_mm_network';
+use utils qw(systemctl zypper_call exec_and_insert_password);
```

Comment:
You don't need `exec_and_insert_password` and use `systemctl` instead `assert_script_run "systemctl`

Reply:
ok, done! Thanks